pkg/hostagent/hostagent.go

```diff
@@ -420,7 +420,11 @@ func (a *HostAgent) Info(_ context.Context) (*hostagentapi.Info, error) {
 
 func (a *HostAgent) startHostAgentRoutines(ctx context.Context) error {
 	if *a.instConfig.Plain {
-		logrus.Info("Running in plain mode. Mounts, port forwarding, containerd, etc. will be ignored. Guest agent will not be running.")
+		if len(a.instConfig.PortForwards) > 0 {
+			logrus.Info("Running in plain mode. Mounts, containerd, etc. will be ignored. Guest agent will not be running. Port forwarding is enabled via static SSH tunnels.")
```

Comment:
This is confusing, portForwads in the yaml must not be used in plain mode. Are we overriding portForwards with the static port forwarding from the command line?

We need to use another configuration for static port forwarding, otherwise port forwarding in the yaml will be applied in plain mode, which will break users assuming the previous behavior (--plain ignores portForwards in the yaml).